db/migrate/20230727161525_add_api_token_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddApiTokenToUser < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :api_token, :string
+  end
+end
```

Comment:
- [ ] Great work so far 👍 but it would be better if you fix the error coming from this migration 😨 When I tried to run the migrations then all the migrations completed but this one gave some error so kindly fix it 😊

Reply:
Thank you.
I'll get on that